src/main/flight/gps_rescue.c

```diff
@@ -758,7 +777,7 @@ void gpsRescueUpdate(void)
             rescueState.intent.yawAttenuator += rescueState.sensor.gpsRescueTaskIntervalSeconds;
         }
         if (rescueState.sensor.absErrorAngle < 30.0f) {
-            rescueState.intent.pitchAngleLimitDeg = gpsRescueConfig()->maxRescueAngle; // allow pitch
+            rescueState.intent.pitchAngleLimitDeg = gpsRescueConfig()->maxRescueAngle * rescueState.sensor.groundspeedErrorRatioInv; // allow pitch
```

Comment:
Is this really necessary? IMO it will be very confusing for someone reading the code.
At least use `groundspeedErrorRatio` and comment its range/use here

Also absErrorAngle < 30, so `groundspeedErrorRatio` would be quite small (groundspeedErrorRatio is function of absErrorAngle )

Reply:
The absErrorAngle is a heading error, but if the IMU is wrong, the heading error can be tiny, whereas the groundspeed error ratio can be very high.

For example, when the IMU is 180 degrees wrong, as in the example above, and if the quad is drifting away quite fast, the groundspeed error ratio can be say 3 or 4.  Like in the example videos above.  

In this situation, when the quad rotates after the climb, the rotation continues until the heading error is brought to zero.  

Howeve, due to the IMU error, the true heading of the quad is 180 degrees wrong when the measured heading error is zero.  After rotating so that the heading error is 30 degrees (which is the point at which we start to pitch forward), a quad in this situation is pointing 150 degrees away from home.  Over the next half a second, that remaining 30 degrees of error is removed, and then the nose is pointing a full 180 degrees from home.  

In this example, we then have a groundspeedErrorRatio of 3 or 4, even though the heading error is close to zero.